packages/mobile/src/dappkit/DappKitAccountScreen.tsx

```diff
@@ -76,10 +76,13 @@ class DappKitAccountAuthScreen extends React.Component<Props> {
     const { dappName } = route.params.dappKitRequest
     return (
       <SafeAreaView style={styles.container}>
+        <TopBarTextButton
+          title={t('cancel')}
+          onPress={this.cancel}
+          titleStyle={{ color: colors.dark }}
```

Comment:
nit: We try to avoid declaring styles inline with the component, unless necessary (animation or dynamic style).

> - By moving styles away from the render function, you're making the code easier to understand.
> - Naming the styles is a good way to add meaning to the low level components in the render function.

Source: https://reactnative.dev/docs/stylesheet.html

Another reason is about performance (though not the case here).
If the underlying component uses `PureComponent / React.memo`, passing a new style like this will defeat the render optimization when the parent component re-render.